metron-interface/metron-config/scripts/package.json

```diff
@@ -9,7 +9,8 @@
     "http-proxy-middleware": "0.17.4",
     "optimist": "0.6.1",
     "serve-favicon": "2.4.2",
```

Comment:
I'm good with that solution.  It seems like it's primarily testing deps, and @merrimanr may have already validated the still unknown dependencies (is my understanding accurate?).  I'm not sure what more validation (if any) @mattf-horton or anyone else needs/wants to see to be comfortable with the release.